### PR TITLE
Make some GxPlugs cross compile

### PR DESCRIPTION
**Pull request by brummer10**
As requested, here are some GxPlugs updated to be able to build cross platform

### CODE REVIEW

File: plugins/package/gxswitchlesswah/gxswitchlesswah.mk [hunk 1]
```diff
@@ -20,7 +20,6 @@ endef
 
 define GXSWITCHLESSWAH_INSTALL_TARGET_CMDS
 	$(GXSWITCHLESSWAH_TARGET_MAKE) install DESTDIR=$(TARGET_DIR) INSTALL_DIR=/usr/lib/lv2
-	cp -rL $($(PKG)_PKGDIR)/GxSwitchlessWah.lv2/* $(TARGET_DIR)/usr/lib/lv2/GxSwitchlessWah.lv2/
```
Comment (falkTX) [inline plugins/package/gxswitchlesswah/gxswitchlesswah.mk:23]: why did you remove this line? does your repo contain the custom ttl and modgui now?

**Comment by brummer10** [comment]
Ping

**Comment by falkTX** [comment]
Finally taking care of plugins, I am updating these on the store and also adding them to mod-desktop.
Will let you know if any of them fail, thanks!

**Comment by falkTX** [comment]
mod dwarf builds are a success, but duo and duox builds fail.
logs show this:

```
2024-09-27 13:57:57,465 - builder - INFO - In file included from /home/toolchain/aarch64-mod-linux-gnueabi/aarch64-mod-linux-gnueabi/include/c++/4.9.4/cstdint:35:0,
2024-09-27 13:57:57,466 - builder - INFO -                  from CollisionDrive.cpp:7:
2024-09-27 13:57:57,466 - builder - INFO - /home/toolchain/aarch64-mod-linux-gnueabi/aarch64-mod-linux-gnueabi/include/c++/4.9.4/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
2024-09-27 13:57:57,466 - builder - INFO -  #error This file requires compiler and library support for the \
2024-09-27 13:57:57,466 - builder - INFO -   ^
2024-09-27 13:57:57,466 - builder - INFO - makefile:150: recipe for target 'nogui' failed
```

The error here is the lack of C++ compiler standard, it is not specified on the makefile.
So for older toolchains as used in the duo and duox it uses C++98 but newer ones like dwarf default to C++11.

Fix is to add -std=gnu++11 to your CXXFLAGS

I can workaround this for now by placing this on the SSE_FLAGS var, but it is good practice to set this flag so I would consider it a real bug. Future compilers will default to newer C++ versions.
There is a risk of, say as example, your code not being buildable with C++23 and that becoming a default in GCC15 or something.

**Comment by falkTX** [comment]
Some issues with switchlesswah:

- custom mod ttl had version 1.0 while your version doesnt have version so it gets 0.0, this prevents updates from happening
- switchlesswah modgui not installed

**Comment by falkTX** [comment]
nevermind my last comment, you added some custom MOD which needed to be enabled manually

**Comment by falkTX** [comment]
macOS builds fail, due to use of unsupported special attributes

```
In file included from plugin/gx_bajatubedriver.cpp:69:
In file included from ./dsp/bajatubedriver.cc:4:
./dsp/bajatubedriver_neg_table.h:27:67: error: argument to 'section' attribute is not valid for this target: mach-o section specifier requires a segment and section separated by a comma
 static tablebajatubedriver_neg_imp<200> bajatubedriver_neg_table __rt_data = {
                                                                  ^
plugin/gx_bajatubedriver.cpp:31:42: note: expanded from macro '__rt_data'
#define __rt_data __attribute__((section(".rt.data")))
                                         ^
In file included from plugin/gx_bajatubedriver.cpp:69:
In file included from ./dsp/bajatubedriver.cc:5:
./dsp/bajatubedriver_table.h:27:59: error: argument to 'section' attribute is not valid for this target: mach-o section specifier requires a segment and section separated by a comma
 static tablebajatubedriver_imp<200> bajatubedriver_table __rt_data = {
                                                          ^
plugin/gx_bajatubedriver.cpp:31:42: note: expanded from macro '__rt_data'
#define __rt_data __attribute__((section(".rt.data")))
                                         ^
In file included from plugin/gx_bajatubedriver.cpp:69:
./dsp/bajatubedriver.cc:391:6: error: argument to 'section' attribute is not valid for this target: mach-o section specifier requires a segment and section separated by a comma
void __rt_func Dsp::compute_static(int count, FAUSTFLOAT *input0, FAUSTFLOAT *output0, PluginLV2 *p)
     ^
plugin/gx_bajatubedriver.cpp:30:42: note: expanded from macro '__rt_func'
#define __rt_func __attribute__((section(".rt.text")))
                                         ^
```

**Comment by brummer10** [comment]
Looks like it's the wrong revision in use, it should be 
0b4def49125702f6ca2d77bdbd246a59672fb8c1

which fix this issue on macos.

**Comment by brummer10** [comment]
The same is true for the KnightFuzz, it should be revision 
715d6d77864708003628548dad625a3e4653e01c

The SlowGear still miss this fix.

**Comment by falkTX** [comment]
I simply merged as you had in the PR.
Can you make another PR once you have slowgear also ready? thank you

**Comment by brummer10** [comment]
Yes, I'll do.
